Src/WitsmlExplorer.Frontend/components/Modals/ObjectPickerModal.tsx

```diff
@@ -129,19 +136,76 @@ const ObjectPickerModal = (props: ObjectPickerProps): React.ReactElement => {
               paddingBottom: invalidUid(objectUid) ? 0 : "24px"
             }}
           />
-          <div style={{ display: "flex", flexDirection: "row", gap: "1rem", paddingLeft: "0.5rem", paddingBottom: "1rem" }}>
+          <ButtonsContainer>
             <Button onClick={onClear}>Clear</Button>
             <Button onClick={onReset}>Reset</Button>
             <Button onClick={onPaste} disabled={objectReference == null || objectReference.objectUids.length != 1}>
               Paste
             </Button>
-          </div>
+            {includeIndexDuplicatesOption && (
+              <StyledCheckbox
+                colors={colors}
+                label="Include index duplicates"
+                onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                  setCheckedIncludeIndexDuplicates(e.target.checked);
+                }}
+                checked={checkedIncludeIndexDuplicates}
+              />
+            )}
+          </ButtonsContainer>
+          {checkedIncludeIndexDuplicates && (
+            <StyledBanner colors={colors}>
+              <Banner.Icon variant="warning">
+                <Icon name="infoCircle" />
+              </Banner.Icon>
+              <Banner.Message>
+                Include index duplicates: This option only takes effect when servers have different numbers of decimals. It is not recommended to search for index duplicates by
+                default, as it may result in unnecessary mismatches. This feature should only be used in special cases that require investigation of anomalies in the index
+                duplicates.
+              </Banner.Message>
+            </StyledBanner>
+          )}
         </ModalContentLayout>
       }
     />
   );
 };
 
+const ButtonsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 1rem;
+  padding-left: 0.5rem;
+  padding-bottom: 1rem;
+`;
+
+const StyledCheckbox = styled(Checkbox)<{ colors: Colors }>`
+  span {
+    color: ${(props) => props.colors.infographic.primaryMossGreen};
+  }
+  span:hover {
+    background: ${(props) => props.colors.interactive.checkBoxHover};
+  }
+`;
+
+const StyledBanner = styled(Banner)<{ colors: Colors }>`
```

Comment:
I suggest moving duplicate generic styled components somewhere else. Maybe it's most readable to create a 'styled components' folder under 'components', and have  a separate file for each component?

Reply:
This is a great suggestion. I'll create a new issue since it should be done in other parts of the code as well.